utility/validator/module.go

```diff
@@ -0,0 +1,75 @@
+package validator
+
+import (
+	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/shared/modules/base_modules"
+)
+
+const (
+	ValidatorModuleName = "validator"
+)
+
+type ValidatorModule interface {
+	modules.Module
+
+	// TODO: add validator module functions here
+	ValidatorUtility
+}
+
+// TODO_IN_THIS_COMMIT: exists to help with type assertions, drop once you've added your own functions
+type ValidatorUtility interface {
+	Stake()
+}
+
+type validator struct {
+	base_modules.IntegratableModule
+	logger *modules.Logger
+}
+
+// type assertions for validator module
+var (
+	_ ValidatorModule = &validator{}
+	// TODO: add validator module functions here
```

Comment:
As long as you do `_ ValidatorUtility = &validator{}` it'll enforce all the functions